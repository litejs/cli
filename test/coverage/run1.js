
var a = this.x ? 1 : 2
, b = (a) => a + 100
, bb = (a) => ({
	a: 1
})

function c(a, b) {
	return a + C() + C()
	throw "error"
	function C() {
		return b + D() + D()
		function D() {
			return b
		}
	}
}

for (var i = 10; i--; ) {
	c(i, a)
}

