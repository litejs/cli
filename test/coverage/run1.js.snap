
var a = this.x ? 1 : 2
, b = (a) => a + 100;

function c(a, b) {
	return a + b
	throw "error"
}

for (var i = 10; i--; ) {
	c(i, a)
}

